### ISSUE TITLE
bridging error queue leads to failures

### ISSUE THREAD

**Issue by FabianTrottmann** [Particular/NServiceBus.Transport.Bridge#168]
Related to: #160 
Current Scenario:
- messages are bridged between msmq and azure service bus endpoints to migrate endpoints step by step.
- ServiceControl/ServiceInsights is used to monitor audit/error queues on both transports

Goal:
I want to setup only 1 instance in ServiceControl and all audit messages and error message should pop up within this single instance. 
The instance I would prefer to use is the azure service bus instance.

Problem: 
To achieve this I needed to bridge the platform queues (according to this documentation: https://docs.particular.net/nservicebus/bridge/configuration#bridging-platform-queues)
For audit messages this works fine. But bridging the error messages leads to the following exception within the bridge:
```
Message shovel operation failed, message will be moved to poc.bridge.error@mycomputername
    System.Exception: Failed to shovel message for endpoint poc.error with id 'someid' from 
    msmq to azureservicebus ---> System.Exception: No target address mapping could be found for source address: 
    poc.ASBService@mycomputername
```
The problem lies around here:
In MessageShovel line `if (!IsAuditMessage(messageToSend))` makes sure that headers are not transformed if msg is an audit message. If it is an error message, the code tries to get the header value `NServiceBus.ReplyToAddress` which is poc.ASBService@mycomputername (as it comes from the msmq error queue with machine name) and of course no mapping is found within the EndpointRegistry here: `(targetEndpointAddressMappings.TryGetValue(sourceAddress, out var targetAddress))`
which leads to the exception above.

If I explicitly step over this logic (just as the audit messages), the error message would pop up in service insights, but this leads to another problem: the FailedQ contains again the fully qualified name (poc.ASBService@mycomputername) and that message cannot be retried, because azure service bus does not have such a queue (it only poc.ASBService).

Is the bridging of the error queue not a supported use case?

**Comment by andreasohlund** [comment]
I think you're hit by https://github.com/Particular/NServiceBus.Transport.Bridge/issues/122

We're actively working on a fix for this. cc @SzymonPobiega @dvdstelt 

In short:

We'll stop translating the ReplyToAddress but keep translating the FailedQ so that retries still work

**Comment by andreasohlund** [comment]
Duplicate of #122 which will be released as part of `1.0.0`
